docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -119,15 +119,17 @@ kotlin {
 The `pod()` function call adds a CocoaPods dependency to the Pod built from this project. Each dependency requires
 a separate function call.
 
-You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version` and
-`source` of the library, in its configuration block.
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
```

Comment:
It's also available in the configuration block. So I would add it to the table bellow. Actually all parameters apart from name of the pod are configurable in it.

Reply:
Thank you! moved to the table